<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-LGOJ7E9" failures="1" tests="6" name="Testcases.ReservedKeywords" time="168.804" errors="0" timestamp="2022-09-19T12:26:11 IST" skipped="0">
  <testcase classname="Testcases.ReservedKeywords" name="Login" time="6.915"/>
  <system-out/>
  <testcase classname="Testcases.ReservedKeywords" name="enterKeywordInJournal" time="9.056"/>
  <system-out/>
  <testcase classname="Testcases.ReservedKeywords" name="checkAdmin" time="29.631"/>
  <system-out/>
  <testcase classname="Testcases.ReservedKeywords" name="AdminLogin" time="43.622"/>
  <system-out/>
  <testcase classname="Testcases.ReservedKeywords" name="checkContactPersonEMail" time="51.920"/>
  <system-out/>
  <testcase classname="Testcases.ReservedKeywords" name="checkPatientsEmail" time="27.660">
    <failure message="Failed due to email expected [Resources and Help from MamaLift] but found [CURIO Notification]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Failed due to email expected [Resources and Help from MamaLift] but found [CURIO Notification]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at PageObjects.AdminKeywords.checkPatientEmail(AdminKeywords.java:146)
at Testcases.ReservedKeywords.checkPatientsEmail(ReservedKeywords.java:78)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkPatientsEmail -->
  <system-out/>
</testsuite> <!-- Testcases.ReservedKeywords -->
